skimage/feature/brief.py

```diff
@@ -64,18 +64,18 @@ class BRIEF(DescriptorExtractor):
     >>> import numpy as np
     >>> square1 = np.zeros((8, 8), dtype=np.int32)
     >>> square1[2:6, 2:6] = 1
-    >>> square1
+    >>> square1  # doctest: +SKIP
```

Comment:
well as the mentioned PR bove, why are we skiing he tests?

Reply:
@Borda see https://github.com/scikit-image/scikit-image/pull/3854#discussion_r279253258.